pages/SettingsScreen.js

```diff
@@ -2,16 +2,16 @@
 // https://aboutreact.com/react-native-navigation-drawer/
 
 import React, { Component } from 'react';
-//import react in our code.
 import { StyleSheet, View, Text } from 'react-native';
-// import all basic components
+import { Auth } from 'aws-amplify';
+import { Button } from 'react-native-elements';
  
-export default class Screen3 extends Component {
+export default class Settings extends Component {
   //Screen3 Component
   render() {
     return (
       <View style={styles.MainContainer}>
-        <Text style={{ fontSize: 23 }}> Screen 3 </Text>
+        <Button title="Sign out" onPress={() => Auth.signOut()} />
```

Comment:
Is there a cool icon we could add to this button?

Reply:
Yea probably, i'll look